src/__tests__/__snapshots__/canonicalFields.int.test.ts.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I am still deciding whether I like this approach. Maybe after poking at it for a bit locally I will have more useful feedback.

Reply:
My initial reply was gonna be "this is the lazy option so you're probably right that I should do better and put it in the test directly"

BUT it also might be valuable to say "this is the actual document generated by the call"...

BUT it's probably more valuable to say "the call returned valid JSON that, when parsed, resulted in this expected object".

Which is a long way of saying that yeah I should probably do better and put it in the test directly ;)

Reply:
I guess if I wanted literal JSON in the test document, it would have to be multi-line (do backticks work for this?), and then in order to assert equality, we would not want actual equality but to parse the expected doc, parse the actual doc, and compare those things for value equality. If that's too awkward, this is good. Regardless, not a show-stopper, you actually have a test!!

Reply:
Ahh got ya; so snapshots are NOT manually written -- they're generated by jest the first time you run the test (we don't control the format).  It's then on us to look at what was written and make sure it reflects what we're expecting.

> we would not want actual equality

Yeah that's what I intended FWIW -- jest equality checks are value equality not memory equality (if you want literal "is" you would say `expect(blah).toBe(blah)`).

---

I'm gonna try moving away from snapshot and see how it feels.

Reply:
It is useful to have both approaches, I think, both snapshot and manual. When looking at failed tests that were from snapshots/benchmarks I will be more suspicious of those (than the manually-created expectations) because it is more likely (than the manual ones) to have a bad original test. I suppose what I'm saying is manual > snapshot > no tests.